--- vendor/ezsystems/ezpublish-kernel/eZ/Bundle/EzPublishLegacyBundle/SetupWizard/ConfigurationConverter.php    2013-12-19 00:57:37.577410791 +0000
+++ vendor/ezsystems/ezpublish-kernel/eZ/Bundle/EzPublishLegacyBundle/SetupWizard/ConfigurationConverter.php.fix        2013-12-19 01:11:09.059409715 +0000
@@ -65,9 +65,6 @@
         if ( !is_array( $siteList ) || empty( $siteList ) )
             throw new InvalidArgumentException( 'siteList', 'can not be empty' );
 
-        if ( !in_array( $adminSiteaccess, $siteList ) )
-            throw new InvalidArgumentException( "adminSiteaccess", "Siteaccess $adminSiteaccess wasn't found in SiteAccessSettings.AvailableSiteAccessList" );
-
         $settings['ezpublish']['siteaccess']['list'] = $siteList;
         $settings['ezpublish']['siteaccess']['groups'] = array();
         $groupName = $sitePackage . '_group';